pkg/sqlite/load.go

```diff
@@ -104,8 +107,16 @@ func (s *sqlLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error
 	if err != nil {
 		return err
 	}
+	substitutesFor, err := bundle.SubstitutesFor()
+	if err != nil {
+		return err
+	}
+
+	if substitutesFor != "" && !s.enableAlpha {
```

Comment:
is this the behavior we want? I think a warning might be more appropriate instead of stopping with an error

Reply:
discussed offline. there are not a lot of reasonable ways to use a bundle with substitutes-for set without adding it to a catalog that has substitution enabled.